packages/odyssey-react/src/components/Button/index.tsx

```diff
@@ -69,10 +70,18 @@ const Button: FunctionComponent<Props> = (props) => {
   } = props;
 
   const componentClass = useCx(
-    "ods-button",
-    `is-ods-button-${variant}`,
-    `is-ods-button-size-${size}`,
-    { "is-ods-button-full-width": wide }
+    styles.root,
+    {
+      [styles.variantSecondary]: variant === 'secondary',
```

Comment:
This maps our Module classes to our React props, since authors shouldn't need to specify "variantSecondary" as a prop value.